### ISSUE TITLE
Install on linux instructions out of date

### ISSUE THREAD

**Issue by ianhi** [micro-manager/micro-manager.github.io#88]
The instructions in https://micro-manager.org/Linux_installation_from_source_(Ubuntu) have links to the the wrong places and may be out of date in other ways. It would be good to either update these or to redirect to https://github.com/micro-manager/micro-manager/blob/main/doc/how-to-build.md which seems to be more up to date

**Comment by ianhi** [comment]
I would also suggest deleting: https://micro-manager.org/Linux_installation_from_source_(Gentoo) as the repo it links to hasn't been updated in 5 years

**Comment by marktsuchida** [comment]
Agreed. how-to-build.md also needs some updates (will do), but it makes sense to keep detailed build instructions in the code repository rather than the wiki. I'm also planning to write VS2019 instructions for Windows asap.

**Comment by marktsuchida** [comment]
I've updated the main links, and `how-to-build.md` has also been [updated](https://github.com/micro-manager/micro-manager/commit/59ab11ad8b60e6d45b52d1c78ab5efca21c602cd).

The rest involves deleting out-of-date pages (preserving anything not covered elsewhere), pending decision on https://github.com/micro-manager/micro-manager.github.io/issues/94#issuecomment-1058374541.
